### PR TITLE
Add ContentTooLargeResponse class 

### PR DESCRIPTION
**Pull request by nprimo**

### CODE REVIEW

**Comment by nprimo** [comment]
- When content is too large it adds an empty line of values instead of showcasing the expected page (?)
  1. infinite loop in `add-entry.php` (?) - I cannot see where it can get stuck
  2. How is it possible to add an empty entry with a JPEG "corrupted" file??

**Comment by nprimo** [comment]
Now the server display the expected 413 page when the POST request send via form is bigger than the values set in the `Config` default constructor

File: srcs/responses/ConentTooLargeResponse.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#include "ContentTooLargeResponse.hpp"
```
Comment (Hevhove) [inline srcs/responses/ConentTooLargeResponse.cpp:1]: Can you change the name of this cpp file? It has a spelling error.

**Comment by Hevhove** [comment]
Do you have a view on the relation between max body size and file size? I assume max body size is in bytes, yet when I entered a 9kb file, with maxbodysize=10000, I received a content too large response. When increasing to 11000, no problem. Might have something to do with 1000 in a kb versus 1024 in a kb..

**Comment by Hevhove** [comment]
Is it possible also to display another URL on top when this error happens, and also give them a link to return to the main page in the HTML? Would be nice to just have something to click on for better experience

**Comment by nprimo** [comment]
About the relationship between max body size and the size of the file that is trying to be uploaded: I guess the total body size of the `POST` request that is sent by the client to the server is equal to the size of the file + the text of the response (the other text information filled in the form)

About the URL on the top, I will look into that - about the user experience, I don't care too much since the page could be changed in the config file + it's not supposed to be a "good/nice" usable website :)

**Comment by Hevhove** [comment]
It definitely does not include the request/response headers. Content is only for the content after the double CRLF. But okay it should be fine!

**Review by Hevhove — APPROVED** [review]

**Pull request merged**